app/views/katello/api/v2/content_view_versions/base.json.rabl

```diff
@@ -70,7 +70,7 @@ child :environments => :environments do
   end
 
   node :activation_key_count do |env|
-    Katello::ActivationKey.where(:environment_id => env.id).where(:content_view_id => version.content_view_id).count
+    Katello::ActivationKey.in_content_views_and_environments(lifecycle_environments: env, content_views: version.content_view).count
```

Comment:
see my comment above. You've passed both; Will this even work the way it is?

Reply:
updated